drivers/flash/Kconfig.nrf_rram

```diff
@@ -61,7 +61,7 @@ endchoice
 config SOC_FLASH_NRF_TIMEOUT_MULTIPLIER
```

Comment:
this value is not used by the driver

Reply:
https://github.com/zephyrproject-rtos/zephyr/blob/e1c51f9a7228da4bfd8ad53acf89fb8c736dc705/drivers/flash/soc_flash_nrf.h#L37